### PR TITLE
chore(dependencies): upgrade carbon-web-components

### PR DESCRIPTION
**Pull request by asudoh**
### Related Ticket(s)

To `v1.8.0`.

### Changelog

**Changed**

- Upgrades `carbon-web-components` to `v1.8.0`.

### CODE REVIEW

File: packages/web-components/tests/snapshots/dds-locale-modal.md [hunk 1]
```diff
@@ -64,10 +61,7 @@
 >
   <div class="bx--modal-content">
     <dds-expressive-modal-header>
-      <dds-expressive-modal-close-button
-        role="button"
-        tabindex="0"
-      >
+      <dds-expressive-modal-close-button>
```
Comment (jeffchew) [inline packages/web-components/tests/snapshots/dds-locale-modal.md:64]: @asudoh I see in the snapshot that the role/tabindex is gone in the snapshot, wondering if this is intentional?

Comment (asudoh) [reply packages/web-components/tests/snapshots/dds-locale-modal.md:64]: Thank you @jeffchew for bringing this up! The answer is yes, it's intentional. Both of those features are moved to `<button>` in the shadow DOM of `<dds-expressive-modal-close-button>`.

Comment (jeffchew) [reply packages/web-components/tests/snapshots/dds-locale-modal.md:64]: thanks, just confirming!

Side note on this, this means that the snapshots made are shallow? Wondering if we should do deeper shapshots?

**Review by jeffchew — APPROVED** [review]
LGTM!

File: packages/web-components/tests/snapshots/dds-locale-modal.md (hunk 1 above)
Comment (asudoh) [reply packages/web-components/tests/snapshots/dds-locale-modal.md:64]: > Wondering if we should do deeper shapshots?

Only single DOM tree is supported by our toolchain, unfortunately. It means that we can generate only a single shadow DOM tree of a component, not nested (composed) tree.

**Pull request merged**